module3-crud-and-data-models/r3-model-view-controller/README.md

```diff
@@ -0,0 +1,83 @@
+# Model View Controller (MVC)
+
+## Introduction to MVC projects
+
+When beginning a new project, there are several different methods that can be used to set up the overall file structure and flow. One of the most commonly used architectural patterns is called MVC. This is an acronym for `Model`, `View`, `Controller`.
+This pattern is favored because of its alignment with the computer science design principle, Separation of Concerns. By dividing up responsibilities within our file structure, we can encapsulate information to be referred to via abstraction and maintain cleaner codebases.
```

Comment:
bold, or include a link, to something about separation of concerns

Reply:
"By dividing up responsibilities within our file structure, we can encapsulate information to be referred to via abstraction and maintain cleaner codebases." 

also this needs to be made a much better explanation.